mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -267,6 +267,68 @@ export class CompareRunView extends Component {
     );
   }
 
+  renderTimeRows(colWidthStyle) {
+    const getTimeAttributes = (runInfo) => {
+      const startTime = runInfo.getStartTime();
+      const endTime = runInfo.getEndTime();
+      return {
+        runUuid: runInfo.run_uuid,
+        startTime: startTime ? Utils.formatTimestamp(startTime) : '(unknown)',
+        endTime: endTime ? Utils.formatTimestamp(endTime) : '(unknown)',
+        duration: startTime && endTime ? Utils.getDuration(startTime, endTime) : '(unknown)',
+      };
+    };
+    const runTimeAttributes = this.props.runInfos.map(getTimeAttributes);
+    const renderCell = ([key, value]) => (
+      <td className='data-value' key={key} style={colWidthStyle}>
+        <Tooltip
+          title={value}
+          color='gray'
+          placement='topLeft'
+          overlayStyle={{ maxWidth: '400px' }}
+          mouseEnterDelay={1.0}
+        >
+          {value}
+        </Tooltip>
+      </td>
+    );
+
+    return (
+      <>
+        <tr>
+          <th scope='row' className='head-value sticky-header' style={colWidthStyle}>
+            <FormattedMessage
+              defaultMessage='Start Time:'
+              // eslint-disable-next-line max-len
+              description='Row title for the start time of runs on the experiment compare runs page'
+            />
+          </th>
+          {runTimeAttributes.map(({ runUuid, startTime }) => [runUuid, startTime]).map(renderCell)}
+        </tr>
+        <tr>
+          <th scope='row' className='head-value sticky-header' style={colWidthStyle}>
+            <FormattedMessage
+              defaultMessage='End Time:'
+              // eslint-disable-next-line max-len
+              description='Row title for the end time of runs on the experiment compare runs page'
+            />
+          </th>
+          {runTimeAttributes.map(({ runUuid, endTime }) => [runUuid, endTime]).map(renderCell)}
+        </tr>
+        <tr>
+          <th scope='row' className='head-value sticky-header' style={colWidthStyle}>
+            <FormattedMessage
+              defaultMessage='Duration:'
+              // eslint-disable-next-line max-len
+              description='Row title for the duration of runs on the experiment compare runs page'
+            />
+          </th>
+          {runTimeAttributes.map(({ runUuid, duration }) => [runUuid, duration]).map(renderCell)}
+        </tr>
+      </>
+    );
+  }
```

Comment:
Can we dry the duplicate code out?

We can create an array with all the configs and loop over the array to render each table row. What do you think?


example:
```
const tableCells = [
   {
       translation = <FormattedMessage
              defaultMessage='Start Time:'
              // eslint-disable-next-line max-len
              description='Row title for the start time of runs on the experiment compare runs page'
            />
   },
   {
         translation =   <FormattedMessage
              defaultMessage='End Time:'
              // eslint-disable-next-line max-len
              description='Row title for the end time of runs on the experiment compare runs page'
            />
   }
   ....
]
```


usage:

```
tableCells.map((cell) => {
<tr>
          <th scope='row' className='head-value sticky-header' style={colWidthStyle}>
            {cell.translation}
          </th>
          {runTimeAttributes.map(({ runUuid, startTime }) => [runUuid, startTime]).map(renderCell)}
        </tr>

})
```

Reply:
sounds good!